starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -172,9 +172,13 @@ class AccountToBeDeployedDetailsFactory:
     class_hash: int
     fee_contract: Contract
 
-    async def get(self) -> AccountToBeDeployedDetails:
+    async def get(
+        self, *, class_hash: Optional[int] = None, argent_calldata: bool = False
```

Comment:
could it be class_hash = self.class_hash?

Reply:
Nope, `self` cannot be accessed before the method body